src/test/java/com/blogify/blogapi/integration/conf/MockData/UserMockData.java

```diff
@@ -85,6 +86,24 @@ public static User manager1() {
         .categories(List.of())
         .isFollowed(null);
   }
+  public static User client3() {
```

Comment:
not a good namen use somme thing like "userToCreate" it''s more logic

look what i was doing with "  @PostMapping("/posts/{postId}/reaction") "

i did it to give u exampl of thing that u have to do